demos/c_api_minimal_app/README.md

```diff
@@ -99,3 +99,65 @@ And then execute the alternative make target:
 cd demos/c_api_minimal_app
 make all_docker
 ```
+
+# Capi Benchmark
+
+This guide shows how to perform benchmark tests using OVMS's CAPI. 
+
+Clone OpenVINO™ Model Server GitHub repository and go to the top directory.
+```bash
+git clone https://github.com/openvinotoolkit/model_server.git
+cd model_server
+```
+Build the tool
+```bash
+bazel build //src:capi_benchmark
```

Comment:
does it work outside of the build docker image? We used to have a makefile to build the app without minimal tooling